extension/src/popup/constants/memoTypes.ts

```diff
@@ -2,4 +2,5 @@ export enum MEMO_TYPES {
   MEMO_ID = "memoId",
```

Comment:
I think this type can be replaced with the stellar-base equivalent
https://github.com/stellar/js-stellar-base/blob/cf370f4e627e2b0e296ceceaebd0164dede396bd/types/index.d.ts#L204